lib/crawly/fetchers/fetcher.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Crawly.Fetchers.Fetcher do
+  @moduledoc """
+  A behavior module for defining Crawly Fetchers
+
+  A fetcher is expected to implement a fetch callback which should return a
+  Crawly.Response
```

Comment:
Perhaps add in that it shoukd receive a Crawly.Request (and return a Crawly. Response)

Reply:
Done!